aicsimageio/readers/tiff_reader.py

```diff
@@ -130,7 +130,11 @@ def _read_delayed(self) -> da.core.Array:
             # Combine length of scenes and operating shape
             # Replace YX dims with empty dimensions
             operating_shape = (len(scenes), *operating_shape)
-            operating_shape = operating_shape[:-2] + (1, 1)
+            if scenes[0].keyframe.samplesperpixel != 1:
+                # if it's a multichannel (RGB) we need to pull in the channels as well
```

Comment:
multichannel is not the right word here.  it's multi-sample

Reply:
They are being mapped to channels. Honestly, that's how I think of them. Multi-sample seems unique to tifffile and not a gerenal concept at least to me. If I'm wrong I'm happy to change it but I do want it to be generally interpretable.

Reply:
samples to me seems like a similar implementation that CZI has for BGR images.

On a high level note, there may be potential value in keeping a document that is basically like "PFF-to-open-standard-dimension-names"... I think we actually use "Scene" when OME uses "Image" -- this could be a place for us to feedback into the OME model and ask how to handle "sampled" images.

Reply:
Yeah, under normal circumstances R/G/B *are* indeed called channels, but it's just too confusing when Channel in volume data means something else.